src/backend/config.ts

```diff
@@ -26,9 +26,9 @@ export function getConfig() {
   return {
     environment: process.env.ENVIRONMENT!,
     hostname: process.env.HOSTNAME || 'localhost',
-    serviceUrl: `${process.env.HOSTNAME === 'localhost' ? 'http' : 'https'}://${
-      process.env.HOSTNAME || 'localhost:3000'
-    }`,
+    serviceUrl: `${
+      process.env.HOSTNAME === 'localhost' || !process.env.HOSTNAME ? 'http' : 'https'
```

Comment:
Nice catch! I was still logged in when made this change and forgot to double check if the SSO flow still work after that :sweat_smile: